backends/qualcomm/passes/recompose_pixel_shuffle.py

```diff
@@ -1,69 +0,0 @@
-# Copyright (c) Qualcomm Innovation Center, Inc.
```

Comment:
This isnt needed anymore?

Reply:
This test case was first introduced by EDSR where existed a mathematically equivalent pixel shuffle implemented with small operators. It seems to be changed back to `nn.PixelShuffle` in the latest example.